src/submodules/io/AgentsIO.jl

```diff
@@ -6,10 +6,12 @@ Submodule containing functionality for serialization and deserialization of mode
 to and from files.
 
 You can save and load agent data to and from CSV files using [`AgentsIO.dump_to_csv`](@ref)
```

Comment:
I think this stuff can go into `api.md` directly. It seems odd that we have sections of the API that describe what's happening all the way, then `AgentsIO` is just at the bottom. As a user: what is that? What does it do?

It'd be better if we made a section called "Checkpointing" or "(De)Serialization" or something like that.